Source/Classes/PINRemoteImageManager.m

```diff
@@ -1361,6 +1502,40 @@ - (BOOL)materializeAndCacheObject:(id)object
     return [self materializeAndCacheObject:object cacheInDisk:diskData additionalCost:additionalCost maxAge:nil url:url key:key options:options outImage:outImage outAltRep:outAlternateRepresentation];
 }
 
+- (BOOL)materializeAndCacheObject:(id)object
+                      cacheInDisk:(NSData *)diskData
+                   additionalCost:(NSUInteger)additionalCost
+                              url:(NSURL *)url
+                              key:(NSString *)key
+                          options:(PINRemoteImageManagerDownloadOptions)options
+                     cacheOptions:(PINRemoteImageManagerCacheOptions)cacheOptions
+                         outImage:(PINImage **)outImage
+                        outAltRep:(id *)outAlternateRepresentation
+{
+    return [self materializeAndCacheObject:object cacheInDisk:diskData additionalCost:additionalCost maxAge:nil url:url key:key options:options cacheOptions:cacheOptions outImage:outImage outAltRep:outAlternateRepresentation];
+}
+
+- (BOOL)materializeAndCacheObject:(id)object
+                      cacheInDisk:(NSData *)diskData
+                   additionalCost:(NSUInteger)additionalCost
+                           maxAge:(NSNumber *)maxAge
+                              url:(NSURL *)url
+                              key:(NSString *)key
+                          options:(PINRemoteImageManagerDownloadOptions)options
+                         outImage:(PINImage **)outImage
+                        outAltRep:(id *)outAlternateRepresentation
+{
+    return [self materializeAndCacheObject:object
+                               cacheInDisk:diskData
+                            additionalCost:additionalCost
+                                    maxAge:nil
+                                       url:url
+                                       key:key
+                                   options:options
+                              cacheOptions:PINRemoteImageManagerCacheOptionsMemory | PINRemoteImageManagerCacheOptionsDisk
```

Comment:
What do you think about adding a `PINRemoteImageManagerCacheOptionsMemory` default option, which can be set to be `PINRemoteImageManagerCacheOptionsMemory | PINRemoteImageManagerCacheOptionsDisk`? I think it would make it easier in the future if we needed to make method calls that use the default behavior or if we ever wanted to change the default behavior altogether

Reply:
Changing the default behavior would affect the existing users of the framework. Thats why I am keeping both as the default.

Reply:
Oh yea I wasn’t suggesting to change the default behavior. Sorry, just realized a typo in my comment. I meant to suggest a PINRemoteImageManagerCacheOptionDefault Option, whoa value would be PINRemoteImageManagerCacheOptionMemory | PINRemoteImageManagerCacheOptionDisk